src/SFA.DAS.LevyTransferMatching.Api/Controllers/OpportunityController.cs

```diff
@@ -123,7 +123,12 @@ public async Task<IActionResult> Apply(long accountId, int opportunityId, [FromB
                 EmailAddresses = request.EmailAddresses,
                 BusinessWebsite = request.BusinessWebsite,
                 UserId = request.UserId,
-                UserDisplayName = request.UserDisplayName
+                UserDisplayName = request.UserDisplayName,
```

Comment:
I'm unclear as to what these properties are here; I can't see any corresponding properties in the applicable model in the UI layer - and I wouldn't expect to see them. When an application is made, it is the inner api that determines the matching criteria for the application; it needn't be given these values.

Reply:
changes made as requested